images/management/commands/import-tandemvault-images.py

```diff
@@ -0,0 +1,531 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from images.models import *
+
+import math
+import logging
+import timeit
+import datetime
+
+import requests
+from progress.bar import Bar
+from dateutil.parser import *
+import boto3
+
+
+class Command(BaseCommand):
+    help = 'Imports image assets from UCF\'s Tandem Vault instance.'
+
+    progress_bar                = Bar('Processing')
+    source                      = 'Tandem Vault'
+    auto_tag_source             = 'AWS Rekognition'
+    auto_tag_blacklist          = [
+        'human',
+        'apparel',
+        'clothing',
+        'skin',
+        'dating',
+        'photo',
+        'photography',
+        'mammal'
+    ]
+    auto_tag_translations       = {
```

Comment:
Can we make this something we can control within the search service, so we don't have to deploy code to add a tag_translation?

Reply:
I'll add a new issue to look into that 👍